src/tactic/mk_opaque.lean

```diff
@@ -0,0 +1,21 @@
+import tactic.core
+
+namespace tactic
+
+setup_tactic_parser
+
+/-- `mk_opaque x y z`, with `x`, `y`, `z` local definitions, transforms them into
+normal local constants -/
+meta def interactive.mk_opaque (ns : parse ident*) : tactic unit :=
+ns.mmap' $ λ n,
+do h ← get_local n,
+   n ← revert h,
```

Comment:
This fails if there are dependencies, right?

Reply:
How so?

Reply:
Ah ok, I missed the `intron n` at the end.  A test for this would be nice.

Reply:
Yes, could you add tests for when something depends on the variable being made opaque? One should fail if there's a dependency on the value.